src/main/java/org/gitlab4j/api/NotesApi.java

```diff
@@ -166,10 +166,28 @@ public Note createIssueNote(Object projectIdOrPath, Long issueIid, String body)
      * @throws GitLabApiException if any exception occurs
      */
     public Note createIssueNote(Object projectIdOrPath, Long issueIid, String body, Date createdAt) throws GitLabApiException {
+        return (createIssueNote(projectIdOrPath, issueIid, body, null, false));    }
+
+    /**
+     * Create a issues's note.
+     *
+     * @param projectIdOrPath the project in the form of an Long(ID), String(path), or Project instance
+     * @param issueIid the issue IID to create the notes for
+     * @param body the content of note
+     * @param createdAt the created time of note
+     * @param internal whether the note shall be marked 'internal'
+     * @return the created Note instance
+     * @throws GitLabApiException if any exception occurs
+     */
+    public Note createIssueNote(Object projectIdOrPath, Long issueIid, String body, Date createdAt, boolean internal) throws GitLabApiException {
 
         GitLabApiForm formData = new GitLabApiForm()
                 .withParam("body", body, true)
-                .withParam("created_at", createdAt);
+                .withParam("created_at", createdAt)
+        ;
+        if (internal) {
+        	formData.withParam("internal", "true");
+        }
```

Comment:
No need to check for the internal value. If you use `Boolean` the `withParam` adds it only if not null.

```suggestion
                .withParam("created_at", createdAt)
                .withParam("internal", internal);
```